electrode-ota-server-routes-accesskeys/src/index.js

```diff
@@ -2,31 +2,32 @@ import {wrap} from 'electrode-ota-server-util';
 import diregister from "electrode-ota-server-diregister";
 export const register = diregister({
     name: 'accessKeysRoute',
-    dependencies: ['electrode:route', 'ota!account', 'ota!scheme']
-}, (options, route, acf) => {
+    dependencies: ['electrode:route', 'ota!account', 'ota!logger', 'ota!scheme']
+}, (options, route, acf, logger) => {
 
     const {addAccessKey, listAccessKeys, updateAccessKey, removeAccessKey, account} = wrap(acf);
 
     route([
         {
             method: 'GET',
             path: '/account',
-
             config: {
                 handler (request, reply) {
+                    logger.info({ req: request }, "get account request");
```

Comment:
we should list specific fields we're interested in logging instead of using the entire request object.

Reply:
will change this. originally was using bunyan which pared down the fields to something mor manageable, but bunyan didn't end up working out.